<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>CVE Feed for OpenIDC -- pyoidc</title>
    <link>https://raw.githubusercontent.com/deepseas/cvelistV5monitor/main/feeds/openidc/pyoidc.rss</link>
    <description>The latest CVEs for OpenIDC -- pyoidc</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Sun, 04 Aug 2024 17:22:04 +0000</lastBuildDate>
    <ttl>60</ttl>
    <item>
      <title>CVE-2020-26244|2024-08-04T15:56:04.228Z -- openidc -- pyoidc
</title>
      <link>https://www.cve.org/CVERecord?id=CVE-2020-26244</link>
      <description>Python oic is a Python OpenID Connect implementation. In Python oic before version 1.2.1, there are several related cryptographic issues affecting client implementations that use the library. The issues are: 1) The IdToken signature algorithm was not checked automatically, but only if the expected algorithm was passed in as a kwarg. 2) JWA `none` algorithm was allowed in all flows. 3) oic.consumer.Consumer.parse_authz returns an unverified IdToken. The verification of the token was left to the discretion of the implementator. 4) iat claim was not checked for sanity (i.e. it could be in the future). These issues are patched in version 1.2.1.</description>
      <guid isPermaLink="false">CVE-2020-26244|2024-08-04T15:56:04.228Z</guid>
      <pubDate>Wed, 02 Dec 2020 20:05:15 +0000</pubDate>
    </item>
  </channel>
</rss>
